addons/portal/static/src/scss/portal.scss

```diff
@@ -510,20 +506,39 @@ form label {
             opacity: 1;
         }
     }
-
-    .o_portal_message_internal_off {
-        .btn-danger {
-            display: none;
-        }
+    .o_portal_chatter_attachment_size {
+        max-width: 180px;
+        min-width: 100px;
```

Comment:
why not 27 ?

Reply:
Because it works ^^. I will create variables to compute the length of the document-type icon + 2* the little cross button to delete the document (which wil approximately be 100px, aka the min-width), and a variable which will be max(max_fixed_page_width/arbitrary_number, min-width) where max_fixed_page_width is 1140px if my memory is correct (hardcoded value for portal pages)

Reply:
I removed this class to return to a "grid" system with attachments that can take up to 2 "columns" (bootstrap trick) depending on the length of the name

Reply:
Mec, 11pm.. please